tap_zendesk_chat/__init__.py

```diff
@@ -58,8 +58,10 @@ def sync(ctx):
     currently_syncing = ctx.state.get("currently_syncing")
     start_idx = streams_.all_stream_ids.index(currently_syncing) \
         if currently_syncing else 0
-    stream_ids_to_sync = [cs.tap_stream_id for cs in ctx.catalog.streams
-                          if cs.is_selected()]
+    stream_ids_to_sync = [
```

Comment:
The changes in this file don't seem relevant. It's probably fine in this case (since it doesn't look like these changes will affect anything), but in general it's good to leave out any changes that aren't relevant to what you are fixing.